src/libs/actions/MapboxToken.js

```diff
@@ -46,11 +46,16 @@ const hasTokenExpired = () => moment().isAfter(currentToken.expiration);
 
 const clearToken = () => {
     console.debug('[MapboxToken] Deleting the token stored in Onyx');
-
     // Use Onyx.set() to delete the key from Onyx, which will trigger a new token to be retrieved from the API.
     Onyx.set(ONYXKEYS.MAPBOX_ACCESS_TOKEN, null);
 };
 
+const fetchToken = () => {
+    console.debug('[MapboxToken] Token does not exist so fetching one');
```

Comment:
should we keep this one?

Reply:
@hayata-suenaga you mean console log or this `fetchToken` method?

Reply:
yes I don't think we have debug logs in our codebase?

Reply:
we have a debug log for onyx updates, pusher events, network queue

Reply:
let me know if you want to remove here let me update.

Reply:
@ntdiary can you confirm ⬆️ ?

Reply:
ah I don't think no, but we can remove this one right?

Reply:
updated @hayata-suenaga